content/Software Support/Installation/Add-libraries-to-Arduino-IDE.md

```diff
@@ -5,31 +5,29 @@ id: 5145457742236
 
 Libraries are a collection of code that provide extra functionality for use in sketches, for example when working with hardware or manipulating data.
```

Comment:
```suggestion
Libraries are a collection of code that provide extra functionality for use in sketches, for example when working with hardware or manipulating data. Learn how to add libraries to Arduino IDE.
```
This can still be used to set the context of the article in the introduction.